packages/main/src/components/AnalyticalTable/hooks/useRowSelectionColumn.tsx

```diff
@@ -60,40 +67,44 @@ export const useRowSelectionColumn: PluginHook<{}> = (hooks) => {
         // The cell can use the individual row's getToggleRowSelectedProps method
         // to the render a checkbox
         // eslint-disable-next-line react/prop-types,react/display-name
-        Cell: ({ row }) => {
+        Cell: (instance) => {
+          const { row, webComponentsReactProperties, dispatch, toggleRowSelected, selectedFlatRows } = instance;
+          const handleCellClick = (e) => {
+            if (TableSelectionBehavior.ROW_SELECTOR === selectionBehavior) {
+              toggleSelected(e, row, webComponentsReactProperties, dispatch, toggleRowSelected, selectedFlatRows);
```

Comment:
I would rather pass the whole instance into this method than destructing all elements. Would be cleaner to read.